CONTRIBUTING.md

```diff
@@ -455,6 +455,34 @@ Some important commands to know about:
 
 This module can be used to mock module variables, environment variables etc. Check out the official [monkeypatch documentation](https://docs.pytest.org/en/6.2.x/monkeypatch.html).
 
+## How to add new modules to the API
+
+## Opening and moving between branches
+
+To open a new branch:
+
+```bash
+git branch branch_name
```

Comment:
This should be:

When beginning implementation on an issue, first create a branch off of develop:

```bash
git branch branch_name develop
```